### PR TITLE
add *_tesimv and other full text search fields, remove unused *_si and *_sim

### PR DESCRIPTION
**Pull request by ndushay**
also tidy up schema.xml and adjust solrconfig so full text field matches can be weighted differently than metadata field matches in relevancy scores.

connects to sul-dlss/spotlight#1175

### CODE REVIEW

File: solr_conf_4_testing/schema.xml [hunk 1]
```diff
@@ -235,8 +235,6 @@
     <dynamicField name="*_facet" type="string" stored="true" indexed="true" multiValued="true" />
     <dynamicField name="*_display" type="string" stored="true" indexed="false" multiValued="true"/>
 -->
-    <dynamicField name="*_si" type="string" stored="true" indexed="true" omitNorms="true" />
```
Comment (cbeer) [inline solr_conf_4_testing/schema.xml:238]: :+1: to removing these redundant fields. Before copying these to sul-solr, though, we ought to double check that no data has been indexed into them.

Comment (ndushay) [inline solr_conf_4_testing/schema.xml:238]: I did a pretty decent check.  No such SearchWorks fields.  No spotlight-dor-resources or other spotlight/ exhibits / gdor-indexer gem code.  Checked an old exhibits Solr on sul-solr and feigenbaum.

Comment (cbeer) [inline solr_conf_4_testing/schema.xml:238]: :+1: perfect. I wasn't sure if you looked through the other exhibits.

Comment (ndushay) [inline solr_conf_4_testing/schema.xml:238]: certainly not ALL of them.  But i figured if the exhibits/spotlight/ indexing  code didn't have it, it probably wouldn't be in any of them, and confirmed with 2 or 3 exhibits' Solr schema browser.

Comment (cbeer) [inline solr_conf_4_testing/schema.xml:238]: Ok. I'll double-check, at least, the exhibits that have gone to production.

**Pull request merged**